proposals/p2006.md

```diff
@@ -375,27 +646,23 @@ Some examples of the immutable view use case are provided below. These include
 static objects.
 
 ```
-void SomeFunction(const int &id) {
-  // Here `id` is an immutable view of some value provided by the caller.
-}
 
-void OtherFunction(...) {
-  // ...
+void SomeFunction(const int &id) { // Here `id` is an immutable view of some
+value provided by the caller. }
 
-  const int &other_id = <some-runtime-expression>;
+void OtherFunction(...) { // ...
 
-  // Cannot mutate `other_id` here either, it is just a view of the result of
-  // `<some-runtime-expression>` above. But we can pass it along to another
-  // function accepting an immutable view:
-  SomeFunction(other_id);
+const int &other_id = <some-runtime-expression>;
 
-  // We can also pass ephemeral values:
-  SomeFunction(other_id + 2);
+// Cannot mutate `other_id` here either, it is just a view of the result of //
+`<some-runtime-expression>` above. But we can pass it along to another //
+function accepting an immutable view: SomeFunction(other_id);
+
+// We can also pass ephemeral values: SomeFunction(other_id + 2);
+
+// Or values that may be backed by read-only memory: static const int fixed_id =
+42; SomeFunction(fixed_id); }
```

Comment:
Looks like prettier autoformat got confused and garbled the formatting here and below.